src/Tools/Microsoft.CodeAnalysis.Toolset.Open/Targets/GenerateAssemblyInfo.targets

```diff
@@ -64,7 +64,7 @@
           Inputs="$(MSBuildThisFileFullPath)"
           Outputs="$(GeneratedAssemblyInfoFile)"
           DependsOnTargets="PrepareForBuild"
-          Condition="'$(Language)' == 'C#' or '$(Language)' == 'VB'">
+          Condition="('$(Language)' == 'C#' or '$(Language)' == 'VB') and '$(OS)' == 'Windows_NT' ">
 
     <WriteCodeFragment AssemblyAttributes="@(AssemblyVersionAttribute)"
                        Language="$(Language)"
```

Comment:
There is a bug in the mono implementation of the VB CodeDom Code Generator that causes this to not compile.  I have a patch in hand to fix it but until I can get the necessary approval for that patch I just disabled this portion of the build.